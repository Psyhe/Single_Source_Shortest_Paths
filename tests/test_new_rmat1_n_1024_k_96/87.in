1024 934 943
936 12 139
936 201 2
936 738 145
936 512 124
936 128 169
936 32 82
936 10 151
936 0 94
937 16 132
940 130 29
941 60 221
