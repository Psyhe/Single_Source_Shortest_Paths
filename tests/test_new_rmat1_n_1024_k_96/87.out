1e+18
1e+18
17
137
1e+18
1e+18
41
228
1e+18
1e+18
